src/pkg/packaging/rpm/dotnet-runtime-deps-rpm_config_centos.7-x64.json

```diff
@@ -3,21 +3,20 @@
     "maintainer_email": "dotnetpackages@dotnetfoundation.org",
     "vendor": ".NET Foundation",
 
-    "package_name": "%SHARED_FRAMEWORK_RPM_PACKAGE_NAME%",
+    "package_name": "%RUNTIME_DEPS_RPM_PACKAGE_NAME%",
     "install_root": "/usr/share/dotnet",
-    "install_doc": "/usr/share/doc/%SHARED_FRAMEWORK_RPM_PACKAGE_NAME%/",
 
-    "short_description": "%SHARED_FRAMEWORK_BRAND_NAME% %SHARED_FRAMEWORK_NUGET_NAME%",
-    "long_description": ".NET Core is a development platform that you can use to build command-line\napplications, microservices and modern websites. It is open source,\ncross-platform and is supported by Microsoft. We hope you enjoy using it!\nIf you do, please consider joining the active community of developers that are\ncontributing to the project on GitHub (https://github.com/dotnet/core).\nWe happily accept issues and PRs.",
-    "homepage": "https://github.com/dotnet/core",
+    "short_description": "%RUNTIME_DEPS_RPM_PACKAGE_NAME% %RUNTIME_DEPS_VERSION%",
+    "long_description": ".NET Core is a development platform that you can use to build command-line applications, microservices and modern websites. This package installs all the system dependencies for Dotnet Core Runtime.",
```

Comment:
`Dotnet Core Runtime` => `.NET Core Runtime`

Reply:
I changed it, but why is that necessary?

Reply:
Because we don't write it "Dotnet" in communication.  It is branded ".NET".

For example, you don't see "Dotnet" anywhere on http://dot.net